launcher/src/gpu.rs

```diff
@@ -0,0 +1,22 @@
+use std::sync::LazyLock;
+
+pub static COMPUTE_CAPABILITY: LazyLock<Option<(isize, isize)>> =
+    LazyLock::new(get_cuda_capability);
+
+fn get_cuda_capability() -> Option<(isize, isize)> {
+    use pyo3::prelude::*;
+
+    let py_get_capability = |py: Python| -> PyResult<(isize, isize)> {
```

Comment:
Really `isize` I'm curious why not `usize`.

Reply:
I was mostly respecting the Python and CUDA types:

https://docs.nvidia.com/cuda/cuda-runtime-api/group__CUDART__DEVICE.html#group__CUDART__DEVICE_1g1bf9d625a931d657e08db2b4391170f0

Reply:
Fine by me, just feels very odd, I wouldn't know what to do with version -10. Having a parsing error there wouldn't shock me tbh.

Reply:
I'll add some validation and then convert it to usize.

Reply:
Added validation and changed types to `usize`.